include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -1255,9 +1253,8 @@ __parallel_find(oneapi::dpl::__internal::__device_backend_tag __backend_tag, _Ex
     auto __keep = oneapi::dpl::__ranges::__get_sycl_range<__par_backend_hetero::access_mode::read, _Iterator>();
     auto __buf = __keep(__first, __last);
 
-    using _TagType =
-        typename ::std::conditional<_IsFirst::value, __parallel_find_forward_tag<decltype(__buf.all_view())>,
-                                    __parallel_find_backward_tag<decltype(__buf.all_view())>>::type;
+    using _TagType = ::std::conditional_t<_IsFirst::value, __parallel_find_forward_tag<decltype(__buf.all_view())>,
```

Comment:
Juts one comment - in case of so big changes it would be good approach to dont change formatting of the existing code, if there is no code changes by a fact, because it makes more difficult a reviewing process.

Reply:
I doesn't seen any changes at this line due comparison with `main` branch.
Could you please explain exactly, what you mean?

Reply:
if there is no diff comparing with the main - fine.